fatiando/gravmag/transform.py

```diff
@@ -141,6 +141,10 @@ def upcontinue(x, y, data, shape, height):
 
     .. note:: x, y, z and height should be in meters.
 
+    .. note:: It is not possible to get the FFT of a masked grid. The default
+        `gridder.interp()` call using minimum curvature will not be suitable. 
+        Use `extrapolate=True` or `algorithm='nearest'`.
```

Comment:
@mtb-za in rst format, you need double `` to format code. You can also link the documentation to other functions like so:

    .. note:: It is not possible to get the FFT of a masked grid. 
        The default :func:`fatiando.gridder.interp` call using minimum curvature will not be suitable.
        Use parameters ``extrapolate=True`` or ``algorithm='nearest'``.